client/js/lounge.js

```diff
@@ -129,6 +129,11 @@ $(function() {
 				feedback.hide();
 			});
 		}
+
+		if (data.token && window.localStorage.setItem("token") !== null) {
```

Comment:
`data.token` will be sent only on `data.success` anyway, so the check won't pass. And yes it should be `getItem`.